pkg/cosign/tuf/client.go

```diff
@@ -213,15 +222,9 @@ func GetRootStatus(ctx context.Context) (*RootStatus, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer t.Close()
 	return t.getRootStatus()
 }
 
-// Close closes the local TUF store. Should only be called once per client.
-func (t *TUF) Close() error {
```

Comment:
Yup, that's right! For longer running processes (like the webhook) there should be a way to refresh the data, but @asraa has some ideas there.

Reply:
For longer running processes we are also refreshing now -- this is handled in `updateClient`: we Update our local in-memory metadata, then syncMetadata to the DB.

Reply:
I'm a little confused why we need the mutex.  Is it because there's a concern that `initializeTUF` will called multiple times by other processes?

Reply:
It shouldn't -- I switched over to `Once` initialization, and reset the `sync.Once` for tests.

The only time it will be called multiple times potentially in parallel is because of the one test outside the TUF pkg that may run in parallel with the TUF tests.